### DISCUSSION TITLE
[Font request]:Heroicons webfonts

### DISCUSSION THREAD

**Discussion by daylmo** [tailwindlabs/heroicons#922] [Ideas]
Is it possible to avail heroicon webfonts?

**Comment by zefir-git** [comment]
By using a font you force the client to download all possible icons, even though you probably don't use all of them (especially not on 1 single page). Fonts may not work on some old browsers and computers. Also, until the font loads, the icons on your page may look like Chinese symbols and other non-printable characters such as ����.

If you really want to use a font, you can check this StackOverflow question: https://stackoverflow.com/q/13278707/7089726

**Reply by daylmo** [reply]
Ok I get it now. Thanks :)
